lib/main_containers.pm

```diff
@@ -299,6 +299,10 @@ sub update_host_and_publish_hdd {
 
 sub load_container_tests {
     my $runtime = get_required_var('CONTAINER_RUNTIMES');
+    if (my $chart = get_var('HELM_CHART')) {
```

Comment:
TLDR: Please move this to the k3s installation part.

I'm against setting those variables here. This makes it impossible to debug and has two major issues:

1. We seldom set variables in test code, users don't expect this to happen
2. The effect of setting those variables is elsewhere, so if someone would need to investigate an issue, one would probably not find where those variables are set.

Let's move them closer to the k3s installation routines, or even better within them, so that anyone ever debugging this code has at least the chance of finding them.

IMHO it's very bad code style to set settings somewhere else than in either a setup/configuration routine of very close to the place where the setting shows the effect. It turns into searching a needle in a haystack otherwise.